tensorboard/components/tf_backend/router.ts

```diff
@@ -28,10 +28,10 @@ export interface Router {
  * Create a router for communicating with the TensorBoard backend. You
  * can pass this to `setRouter` to make it the global router.
  *
- * @param pathPrefix {string=} The base prefix for data endpoints.
+ * @param pathPrefix {string} The base prefix for data endpoints.
  * @param demoMode {boolean=} Whether to modify urls for filesystem demo usage.
  */
-export function createRouter(pathPrefix = 'data', demoMode = false): Router {
+export function createRouter(pathPrefix, demoMode = false): Router {
```

Comment:
Why is the default prefix no longer `"data"`? This is a breaking
change—does it just not happen to break any clients?

Reply:
I vetted all the callsites manually

Reply:
> I vetted all the callsites manually

I mean, this is not possible in principle, right? `createRouter` is a
publicly exported function defined in a module whose Bazel target has
visibility `//visibility:public` in a workspace that we explicitly
encourage people to import, both inside Google and beyond. By all
accounts it is a public API. Are we in the habit of breaking our public
contracts?

I also still don’t understand _why_ you want to make this change.